tests/test_bell_measure_example.py

```diff
@@ -17,12 +17,13 @@
 
 
 import subprocess
+import sys
 
 
 def test_bell_measure_example_call():
     """Test that example script runs and fails at establishing a connection."""
     with subprocess.Popen(
-        ("python", "examples/bell_measure.py", "--cortex_server_url", "https://not.a.real.domain"),
+        (sys.executable, "examples/bell_measure.py", "--cortex_server_url", "https://not.a.real.domain"),
```

Comment:
@kukushechkin, I would be really curious to understand why did the test break on Windows. Any ideas? :-)

Reply:
I guess the the "parent" python environment (comes from tox I guess?) is different from whatever we call by just calling "python" (comes from venv I guess?). This was it seems to be more consistent, it is not any random python, it is the same python that runs the tests.